src/adapter/breakpointPredictor.ts

```diff
@@ -120,10 +121,11 @@ class DirectoryScanner {
           continue;
         }
 
-        let set = sourcePathToCompiled.get(resolvedPath);
+        const normalizedPath = urlUtils.lowerCaseInsensitivePath(resolvedPath);
```

Comment:
This is a new function that returns a lowercase version of the path if we're on a case-insensitive filesystem. I'm using this as hashmap keys here and elsewhere to avoid duplication.